docs/versioned_docs/version-2.11.0/introduction/releases.md

```diff
@@ -107,27 +107,27 @@ OpenEBS v2.9 is another maintenance release before moving towards 3.0 primarily
 OpenEBS is a collection of data engines and operators to create different types of replicated and local persistent volumes for Kubernetes Stateful workloads. Kubernetes volumes can be provisioned via CSI Drivers or using Out-of-tree Provisioners. The status of the various components as of v2.9.0 are as follows:
 
 - CSI Drivers
-  - [Mayastor](https://docs.openebs.io/docs/next/ugmayastor.html) 0.8.0 (beta)
+  - [Mayastor](/docs/user-guides/mayastor) 0.8.0 (beta)
   - [cStor](https://github.com/openebs/cstor-operators) 2.9.0 (beta)
   - [Jiva](https://github.com/openebs/jiva-operator) 2.9.0 (beta)
   - [Local PV ZFS](https://github.com/openebs/zfs-localpv) 1.7.0 (stable)
   - [Local PV LVM](https://github.com/openebs/lvm-localpv) 0.5.0 (beta)
   - [Local PV Rawfile](https://github.com/openebs/rawfile-localpv) 0.4.4 (beta)
   - [Local PV Partitions](https://github.com/openebs/device-localpv) 0.2.0 (alpha)
 - Out-of-tree provisioners
-  - [Jiva](https://docs.openebs.io/docs/next/jiva.html) 2.9.0 (stable)
-  - [Local PV hostpath](https://docs.openebs.io/docs/next/uglocalpv-hostpath.html) 2.9.0 (stable)
-  - [Local PV device](https://docs.openebs.io/docs/next/uglocalpv-device.html) 2.9.0 (stable)
-  - [cStor](https://docs.openebs.io/docs/next/cstor.html) 2.9.0 (beta)
+  - [Jiva](/docs/user-guides/jivaguide) 2.9.0 (stable)
```

Comment:
```suggestion
  - [Jiva](/docs/user-guides/jiva-guide) 2.9.0 (stable)
```
Please change this link to `jiva-guide` since these are two separate words

Reply:
will do that in a separate PR, as the change has to apply in some other files too

Reply:
Ok